Modules/CLI/ResampleDTIVolume/ResampleDTIVolume.cxx

```diff
@@ -1011,21 +1012,28 @@ int Do( parameters list )
     }
   double defaultPixelValue = list.defaultPixelValue;
   // start transform
-  transform->SetMeasurementFrame( measurementFrame ); // This was probably wrong in the old versions
+
     {                                                 // local for memory management: the input image should not stay in
                                                       // memory after we exit this portion of code
     ResamplerTypePointer resampler = ResamplerType::New();
     resampler->SetInput( image );
-    resampler->SetTransform( transform );
     if( list.numberOfThread )
       {
       resampler->SetNumberOfThreads( list.numberOfThread );
       }
     resampler->SetInterpolator( interpol );
-    SetOutputParameters<PixelType>( list, resampler, image );
+    itk::Matrix<double, 3, 3> outputImageDirection ;
+    outputImageDirection = SetOutputParameters<PixelType>( list, resampler, image );
+    if( !hasMeasurementFrame || list.noMeasurementFrame )
+    {
+      itk::Matrix<double, 3, 3> transposeOutputDirection ( outputImageDirection.GetTranspose() ) ;
+      measurementFrame = transposeOutputDirection * measurementFrame ;
+    }
     resampler->SetDefaultPixelValue( static_cast<PixelType>( defaultPixelValue ) );    // Could be set directly in the
                                                                                        // interpolator, but we keep
                                                                                        // ITKResampleImageFilter design
+    transform->SetMeasurementFrame( measurementFrame ); // This was probably wrong in the old versions
```

Comment:
Since you are updating this file, do you know what we should do about this call ? Is the comment ` This was probably wrong in the old versions` still valid ?